pkg/client/client.go

```diff
@@ -0,0 +1,107 @@
+package client
```

Comment:
Couldn't this and `def.go` just be in the root folder? It makes the import statement a little nicer for consumers of the client.

Reply:
Restructuring made to fix this